tdrs-backend/tdpservice/users/test/factories.py

```diff
@@ -57,6 +57,12 @@ class STTUserFactory(BaseUserFactory):
     # were calling the `populate_stt` command.
     stt = None
 
+class AdminSTTUserFactory(STTUserFactory):
```

Comment:
I needed an admin user who didn't have an stt associated with them for testing the issue alex encountered